packaging/greenboot/microshift-running-check.sh

```diff
@@ -25,6 +25,36 @@ function forced_termination() {
     exit 1
 }
 
+# Check preconditions for existence of topolvm deployment.
+# Adapted from MicroShift code.
+#
+# args: None
+# return: 0 if topolvm readiness should be checked, 1 otherwise
+function topolvmShouldBeDeployed() {
+    if ! hash vgs 2>/dev/null; then
+        return 1
+    fi
+
+    # TODO: If lvmd.yaml exists, should we expect topolvm even if the OS doesn't has lvms2 installed?
```

Comment:
I'm not sure how to intepret this comment. Can you explain more please?

Reply:
If `lvmd.yaml` exists, but `lvms2` tools are not installed - which is stronger signal that user wants topolvm?
What if user created `lvmd.yaml` but forgot to install lvms2?

Reply:
So, in this case, would microshift decide to start topolvm and fail?

Reply:
from impl perspective it's just like currently implemented here: if `vgs` is absent, existence of `lvmd.yaml` doesn't have any impact

Reply:
and what I'm wondering is: should we flip it - if `lvms.yaml` is present, but `vgs` is not, then should we fail (and where - here or microshift itself)

Reply:
The check stays as it is - question was about flipping this one and first one (`hash vgs`)